source/includes/client_sdk.md

```diff
@@ -34,10 +40,11 @@
 
 In order to ensure a simple and delightful experience for customers, we offer an integrated widget.
 
-Highline's widget is a front-end UI element that allows users to grant your application access to their work accounts, set up automated payments, and switch deposits directly from their paychecks.
+Highline's widget is a <a class="content-link" href="https://developer.mozilla.org/en-US/docs/Web/Web_Components" target="_blank">Web Component</a> that allows users to grant your application access to their work accounts, set up automated payments, and switch deposits directly from their paychecks.
 
-To use Highline-Link, pass in an object with the parameters described below to the  
-`highlineLink.connect()` function:
+To use Highline-Link, include the <span style='white-space:no-wrap;'>`<highline-link></highline-link>`</span> tag in your HTML document, then include the script tag for the environment you wish to target. 
+
+After these two tags are included on your page, connect to Highline-Link by passing in an object with the parameters described below to the `highlineLink.connect()` JavaScript function.  Once connected open the widget with `highlineLink.open()`.
```

Comment:
Do we need an example for highlineLink.open() e.g., onClickHandler or something?

Reply:
I don't think so, just wanted to go with the basics here.   Onclick isn't necessary to open it, though we could add some javascript guide to help inexperienced devs if needed.

Reply:
Might be worth adding something along the lines of
// `highlineLink.open()` would typically be user-initiated, such as a button click or tap